tests/init/config-initializer.js

```diff
@@ -137,6 +138,21 @@ describe("configInitializer", () => {
                     format: "JSON"
                 };
             });
+            it("should throw error with message when no package.json", async () => {
+                const requireStubs = {
+                    "../../lib/shared/logging.js": log,
+                    "../../lib/init/npm-utils.js": {
+                        ...await esmock("../../lib/init/npm-utils.js", { fs: defineInMemoryFs({}) })
+                    }
+                };
+
+
+                init = await esmock("../../lib/init/config-initializer.js", requireStubs, { });
+
+                assert.throws(() => {
+                    init.initializeConfig();
+                }, "Cannot initialize ESLint outside a node project");
```

Comment:
```suggestion
                }, "A package.json file is necessary to initialize ESLint. Run `npm init` to create a package.json file and try again.");
```

Need to update here as well.